### ISSUE TITLE
Add integration tests

### ISSUE THREAD

**Issue by nilsreichardt** [saibotma/jitsi_meet_wrapper#31]
## Description
I would be great to have an integration test for this package, to be sure it's working as expected. 

I think for the beginning would be a test case where we run the example app, join the meeting, take a screenshot and compare this with the master screenshot great. It covers the integration with the native SDKs and checks if the user is able to join the meeting - and I think this is the most important thing.

We need to be sure that the Jitsi room is empty, because otherwise would be the UI a bit different. So generating a new room ID (like a `uuid4`) for every test would be good.

I already started to implement this integration test (https://github.com/nilsreichardt/jitsi_meet_wrapper/tree/integration-tests). But I'm currently I stuck when I try to compare the screenshot with the master screenshot. So I created a Flutter issue for help: https://github.com/flutter/flutter/issues/103222

**Comment by saibotma** [comment]
Will try to take a look at it the next days.

**Comment by nilsreichardt** [comment]
It's no problem for me to do this, I just wanted to create this issue to inform you / discuss the implementation with you, if needed 👍

**Comment by saibotma** [comment]
Yeah, will try to have a look at your implementation. That's what I wanted to say ;)

**Comment by saibotma** [comment]
I am sorry for not responding, but i currently don't have time to work on this issue.
